apps/DesktopStreamer/MainWindow.h

```diff
@@ -106,6 +110,7 @@ private slots:
     QImage cursor_;
 
     QTimer updateTimer_;
+    QTimer browseTimer_;
```

Comment:
I agree that in the general case ifdefs in headers are bad, but this specific header is not meant to be included anywhere... and since you already introduced an ifdef anyway, it is a good idea to group the browserTimer with the servus. Pimpl is good too, of course.